### DISCUSSION TITLE
Styling a row while certain condition is met

### DISCUSSION THREAD

**Discussion by danielbettech** [icflorescu/mantine-datatable#135] [Q&A]
Hello!

I have the need to style a row based on certain condition. Do you think is possible to implement something like this?

```javascript
rowStyles: ({ completed }) => ({ backgroundColor: completed && '#636363' })
```

**Comment by icflorescu** [comment]
Created issue #152.

**Comment by icflorescu** [comment]
This is possible starting with `v1.7.28` 🎉
TL;DR: Use `rowClassName` or `rowStyle`; they both accept a function like this: `(record, recordIndex) => ...`

**Reply by danielbettech** [reply]
Thanks a lot!
